monday/graphqlclient/client.py

```diff
@@ -1,41 +1,56 @@
-import requests
 import json
 
+import requests
+
+from monday.exceptions import MondayQueryError
+
 
 class GraphQLClient:
     def __init__(self, endpoint):
         self.endpoint = endpoint
         self.token = None
-        self.headername = None
+        self.token_header_name = None
+        self.headers = {}
 
     def execute(self, query, variables=None):
         return self._send(query, variables)
 
-    def inject_token(self, token, headername='Authorization'):
+    def inject_token(self, token, token_header_name='Authorization'):
```

Comment:
will the header name ever be named differently? if not, can probably scrap this bit

Reply:
Done.